lancache-litmus/overlay/var/www/litmus/index.html

```diff
@@ -1,208 +1,697 @@
+            background: #f0fdf4;
+            border-left-color: var(--success-color);
+            color: #166534;
+        }
+
+        .alert-warning {
+            background: #fffbeb;
+            border-left-color: var(--warning-color);
+            color: #92400e;
+        }
+
+        .alert-danger {
+            background: #fef2f2;
+            border-left-color: var(--danger-color);
+            color: #991b1b;
+        }
+
+        .alert .close {
+            position: absolute;
+            top: 1rem;
+            right: 1rem;
+            background: none;
+            border: none;
+            font-size: 1.5rem;
+            cursor: pointer;
+            opacity: 0.6;
+            color: inherit;
+        }
+
+        .alert .close:hover {
+            opacity: 1;
+        }
+
+        /* CDN Cards Grid */
+        .services-header {
+            margin: 2rem 0 1.5rem 0;
+            text-align: center;
+        }
+
+        .services-header h2 {
+            color: var(--text-color);
+            font-size: 2rem;
+            font-weight: 700;
+            margin-bottom: 0.5rem;
+        }
+
+        .services-header p {
+            color: var(--text-color);
+            font-size: 1.1rem;
+            opacity: 0.8;
+        }
+
+        .cdn-grid {
+            display: grid;
+            grid-template-columns: repeat(auto-fill, minmax(320px, 1fr));
+            gap: 1.3rem;
+        }
+
+        .cdn-card {
+			background: var(--background-color);
+			border-radius: 16px;
+			padding: 1.75rem;
+			box-shadow: var(--shadow);
+			border: 2px solid rgba(255,255,255,0.1);
+			position: relative;
+			display: flex;
+			flex-direction: column;
+			transition: transform .3s, box-shadow .3s;
 		}
-	</style>
+
+        .cdn-card:hover {
+            transform: translateY(-4px);
+            box-shadow: var(--shadow-hover);
+        }
+
+        .cdn-card.status-success   { border-top: 6px solid var(--success-color); }
+		.cdn-card.status-warning   { border-top: 6px solid var(--warning-color); }
+		.cdn-card.status-danger    { border-top: 6px solid var(--danger-color); }
+		.cdn-card.status-loading   { border-top: 6px solid rgba(100,100,100,.5); }
+
+        .cdn-header {
+            display: flex;
+            align-items: center;
+            margin-bottom: 1rem;
+            flex-shrink: 0;
+        }
+
+        .cdn-logo {
+            width: 40px;
+            height: 40px;
+            border-radius: 8px;
+            margin-right: 1rem;
+            background: rgba(233, 29, 18, 0.1);
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            font-weight: 700;
+            color: var(--text-color);
+            flex-shrink: 0;
+            border: 1px solid var(--primary-color);
+        }
+
+        .cdn-title {
+            font-size: 1.3rem;
+            font-weight: 700;
+            color: var(--text-color);
+            text-transform: capitalize;
```

Comment:
```suggestion
            font-family: monospace;
```
Many of the service collection names don't look substantially better with initial capitalisation.